### PR TITLE
tconfig: fix setting default value and refine logging

### PR DESCRIPTION
**Pull request by metalefty**

### CODE REVIEW

**Comment by matt335672** [comment]
I'll look at this ASAP. I'm out of time today after the startwm.sh review.

File: xrdp/xrdp_tconfig.c [hunk 1]
```diff
@@ -190,10 +190,10 @@ tconfig_load_gfx_x264_ct(toml_table_t *tfile, const int connection_type,
...
     {
         TCLOG(LOG_LEVEL_WARNING,
-              "x264 param fps_den is not set for connection type [%s], "
-              "adopting the default value [0]",
-              rdpbcgr_connection_type_names[connection_type]);
-        param[connection_type].fps_num = X264_DEFAULT_FPS_DEN;
+              "[x264.%s] fps_den is not set, adopting the default value [%d]",
+              rdpbcgr_connection_type_names[connection_type],
+              X264_DEFAULT_FPS_DEN);
+        param[connection_type].fps_den = X264_DEFAULT_FPS_DEN;
```
Comment (metalefty) [inline xrdp/xrdp_tconfig.c:196]: Here's the point.

**Comment by metalefty** [comment]
Logging is a cosmetic change. The important change is just one line.

**Comment by metalefty** [comment]
Code reformat and force-pushed.

**Review by matt335672 — APPROVED** [review]
Looks fine.

Sorry about the delay.

**Pull request merged**